tests/integration/test_repeat.py

```diff
@@ -49,14 +55,27 @@ def test_array_empty_repeats_valid(repeats):
 
 
 @pytest.mark.parametrize("repeats", ([3, 4], [1, 2, 3]))
-def test_array_empty_repeats_invalid_negative(repeats):
+def test_array_empty_repeats_invalid_negativei(repeats):
     # numpy raises:
     # ValueError: operands could not be broadcast together with shape (0,) (2,)
     # while cunumeric is pass with the result []
     res_num = num.repeat([], repeats)
     assert np.array_equal(res_num, [])
```

Comment:
I think this test can just be deleted? unless we mean to codify this as an intentional and expected difference from numpy

Reply:
yes, deleted.